### PR TITLE
Document how to release helm charts for mixer

### PR DESCRIPTION
**Pull request by Fructokinase**
Add instructions on releasing helm charts. A released mixer chart can then be used by the website helm chart.

### CODE REVIEW

File: deploy/helm_charts/README.md [hunk 1]
```diff
@@ -36,3 +36,29 @@ helm upgrade --install mixer-dev deploy/helm_charts/mixer \
     --set-file kgStoreConfig.bigqueryVersion=deploy/storage/bigquery.version \
     --set-file kgStoreConfig.bigtableImportGroupsVersion=deploy/storage/bigtable_import_groups.version
 ```
+
+## Releasing a helm chart
+
+Releasing a helm chart packages and uploads a helm chart into [AR(Artifact Registry)](https://cloud.google.com/artifact-registry/docs/overview) as a versioned artifact. Once released, the chart will be accessible by the public, given that the access to AR repository is configured.
+
+1.  Skip this step for upgrades. Make sure that an AR docker repository exists. If not, please follow the instructions [here](https://cloud.google.com/artifact-registry/docs/helm) to create one.
```
Comment (shifucun) [inline deploy/helm_charts/README.md:44]: Maybe add a little background for each bullet here. For example, 

1) I am not sure what "upgrades" here means.
3) should "mixer" be the only choice here?

File: deploy/helm_charts/README.md [hunk 2]
```diff
@@ -36,3 +36,47 @@ helm upgrade --install mixer-dev deploy/helm_charts/mixer \
...
     --set-file kgStoreConfig.bigtableImportGroupsVersion=deploy/storage/bigtable_import_groups.version
 ```
+
+## Releasing a helm chart
+
+Releasing a helm chart packages and uploads a helm chart into [AR(Artifact Registry)](https://cloud.google.com/artifact-registry/docs/overview) as a versioned artifact. Once released, the chart will be accessible by the public, given that the access to AR repository is configured.
+
+### Prerequisites
+
+Make sure that the repository exists with the following gcloud command.
```
Comment (shifucun) [inline deploy/helm_charts/README.md:46]: Curious when would a developer need to create a new repo?

File: deploy/helm_charts/README.md [hunk 3]
```diff
@@ -36,3 +36,47 @@ helm upgrade --install mixer-dev deploy/helm_charts/mixer \
...
+ ```
+
+If it does not, please follow the instructions [here](https://cloud.google.com/artifact-registry/docs/helm) to create a new repository.
+
+### Steps
+
+The following steps are for Mixer's server chart, located at `deploy/helm_charts/mixer`.
+    
+1.  Update the version field in Chart.yaml. The version follows the [semantic versioning](https://semver.org/) format.
+2.  Go the specific chart's root directory.
```
Comment (shifucun) [inline deploy/helm_charts/README.md:60]: Go to

**Review by shifucun — APPROVED** [review]
thanks for the update!

File: deploy/helm_charts/README.md (hunk 2 above)
Comment (Fructokinase) [reply deploy/helm_charts/README.md]: Deleted that comment, it was a one time setup and it should always exist.

File: deploy/helm_charts/README.md (hunk 3 above)
Comment (Fructokinase) [reply deploy/helm_charts/README.md]: done, thanks!

**Comment by shifucun** [comment]
Is this PR ready to submit?

**Pull request closed without merging**